testapp/proto/testapp/v1/get.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package testapp.v1;
 
+option go_package = "github.com/polymerdao/monomer/gen/testapp/v1;testappv1";
```

Comment:
Do you know why this is necessary? Do Cosmos SDK proto files have this? Feels like bad practice to have this [when using `buf`](https://protobuf.com/docs/descriptors#resolving-import-locations), but maybe I'm wrong.

Reply:
Yep, all of the cosmos sdk/osmosis proto files use the `option go_package` annotation. It's really unfortunate that the buf/cosmos integration isn't better but I guess it's also pretty useful to be able to explicitly define the package name you want your generated files to have. For the linked example above we need to use the `;` notation as well since only the `v1` package name would be used by default, however we wanted it to be `testappv1` based on the existing imports.

Here are some examples of the `go_package` annotation in the sdk/osmosis:
- [x/bank tx.proto](https://github.com/cosmos/cosmos-sdk/blob/main/x/bank/proto/cosmos/bank/v1beta1/tx.proto#L11)
- [x/auth auth.proto](https://github.com/cosmos/cosmos-sdk/blob/df3b035dd876fff8375beb4380d4cd063361f3fa/x/auth/proto/cosmos/auth/v1beta1/auth.proto#L9)
- [osmosis x/tokenfactory tx.proto](https://github.com/osmosis-labs/osmosis/blob/main/proto/osmosis/tokenfactory/v1beta1/tx.proto#L10)